rmf_demos_panel/README.md

```diff
@@ -1,20 +1,17 @@
-## RMF Demo Panel Installation
+## RMF Demos Panel
+This will describes more about the details of `rmf_demos_panel`. In short, this package uses a simple web server
+to expose the essential rmf topics/services to the user as web endpoints. Currently, this lite implementation is 
+useful for task submission and observe on-going tasks/robots in RMF.
 
-Setup `rmf_demos_panel`
+### RMF Dependencies
+ - `rmf_task_ros2`
+ - `rmf_fleet_msgs`
+ - `rmf_demos_dashboard_resources`
 
-Rosdep will automatically install system version of python3-flask and python3-flask-cors. Yet we will download flask-socketio (5.x) separately via pip since the ubutuntu packaged version is too old.
-```bash
-python3 -m pip install flask-socketio
-```
+### Setup `rmf_demos_panel`
 
-Compilation
 ```bash
-cd ~/rmf_demos_ws
-
-# change the npm prefix according to the path to "rmf_demos_panel/static/"
-npm install --prefix src/rmf/rmf_demos/rmf_demos_panel/rmf_demos_panel/static/
-npm run build --prefix src/rmf/rmf_demos/rmf_demos_panel/rmf_demos_panel/static/
-
+python3 -m pip install flask-socketio
```

Comment:
```suggestion
python3 -m pip install Flask flask-socketio flask-cors
```

Reply:
Or are these installed by `rosdep`?

Reply:
Good catch. Unfortunately, we cant download `flask-socketio` (only) via `rosdep` due to https://github.com/osrf/rmf_demos/issues/202